### PR TITLE
vm: remove crypto-related opcodes

### PR DESCRIPTION
**Pull request by fyrchik**
All cryptography has moved to interops in NEO3.
There is no SHA256 interop RN, but it is to appear later.

### CODE REVIEW

File: pkg/core/interop/crypto/interop.go [hunk 1]
```diff
@@ -9,6 +9,7 @@ import (
 var (
 	ecdsaVerifyID        = emit.InteropNameToID([]byte("Neo.Crypto.ECDsaVerify"))
 	ecdsaCheckMultisigID = emit.InteropNameToID([]byte("Neo.Crypto.ECDsaCheckMultiSig"))
+	sha256ID             = emit.InteropNameToID([]byte("Neo.Crypto.Sha256"))
```
Comment (roman-khimov) [inline pkg/core/interop/crypto/interop.go:12]: Take a look at neo-project/neo#1419, it's post-preview2, but it's a compatible change wrt this hash function addition (secp256k1 is another topic, I'll open an issue for it, but that's post-0.90.0 for us). I think we can add it in this fashion (`SHA256` vs `Sha256`) right into `interops.go`.

Comment (fyrchik) [reply pkg/core/interop/crypto/interop.go:12]: `core` is already to big, the point of moving some interops out was exactly ability to define them out of `core` (`crypto` in this case).

Comment (roman-khimov) [reply pkg/core/interop/crypto/interop.go]: I mean interop name to function binding, not function implementation.

Comment (fyrchik) [reply pkg/core/interop/crypto/interop.go]: Oh, yeah, fixed.

**Comment by roman-khimov** [comment]
Closes #777?

**Pull request merged**